test/output/slim/block_audio.html

```diff
@@ -0,0 +1,28 @@
+<!-- .basic -->
+<div class="audioblock">
+  <div class="content">
+    <audio controls="" src="ocean_waves.mp3">Your browser does not support the audio tag.</audio>
```

Comment:
There should be `controls`, without `=""` (and the same situation with `loop` and some other attributes). This is problem with nokogiri, it doesn’t not support HTML5. :( However, you can manually fix it in generated examples, it can parse it well and assert passes. It’s not a big deal though.